LICENSE

```diff
@@ -0,0 +1,35 @@
+Copyright (C) 2013-2016 Open Source Robotics Foundation, Inc; CA, USA
+Copyright (C) 2016 Janko Petereit <janko.petereit@gmail.com>
```

Comment:
I assume you added this line based on 180aea62e9e98451b8389c2998a830532266593b

As far as I understand copyright minor changes like this don't qualify. It needs to be "substantial" and / or "creative".